<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDev</class>
 <widget class="QWidget" name="addDev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加设备型号</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_pk">
     <property name="text">
      <string>productKey</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>设备名称</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBox_regFunc">
     <item>
      <property name="text">
       <string>写单个寄存器(0X06)+写多个寄存器(0X10)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>写多个寄存器(0X10)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_ps">
     <property name="text">
      <string>productSecret</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_pk"/>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEdit_ps"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>写寄存器模式</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>写线圈模式</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_coilFunc">
     <item>
      <property name="text">
       <string>写单个线圈(0X05)+写多个线圈(0X0F)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>写多个线圈(0X0F)</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
